### DISCUSSION TITLE
Using Amazon FBA for order fulfillment

### DISCUSSION THREAD

**Discussion by jheathco** [solidusio/solidus#4789] [Troubleshooting]
What is the recommended way for using FBA for order fulfillment?  Does this need to be done through the ShipStation extension, or is there a direct integration with Amazon FBA to handle inventory quantities available for shipment, as well as shipments themselves?

**Comment by jarednorman** [comment]
There's not an Amazon FBA extension. I don't know if you could do it through the ShipStation extension. I've only seen stores role their own for this.
